packages/frontend/src/assets/data/aineistoluokka.json

```diff
@@ -0,0 +1,46 @@
+[
```

Comment:
Having a filename be in Finnish is a bit weird. But it is under assets, so it might be okay. Leaving this one to percolate for now 🤔

Reply:
How about material class or category class? Does it tell full meaning of "aineistoluokka"?

Reply:
I can't really say for sure, as I'm no domain expert. But maybe material class would be fine. Too bad `.json` can't have comments...